neutrons/data_workflow
# Remove Dockerfile.autoreducer.himem and update configurations

# Description of the changes
Deleted Dockerfile.autoreducer.himem and refactored docker-compose.yml to use Dockerfile.autoreducer for both standard and high memory configurations. Updated Dockerfile.webmonchow to install webmonchow version 1.0.1 for improved stability.

Check all that apply:

**References:**
- Links to IBM EWM items: [7035](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20%28Change%20Management%29#action=com.ibm.team.workitem.viewWorkItem&id=7035)

# Manual test for the reviewer
Install the local deployment
```
make clean
make all
make locadev/up
```
Enter container `data_workflow-feature-autoreducer_himem-1` and verify that file `/etc/autoreduce/post_processing.conf` in this container corresponds to file `tests/configuration/post_process_consumer.himem.conf` in the repo

# Check list for the reviewer
